spec/order_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'order'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.